sycl/cmake/modules/AddSYCLDeviceTarget.cmake

```diff
@@ -0,0 +1,24 @@
+macro(add_sycl_executable ARG_TARGET_NAME)
```

Comment:
I would name it `AddSYCLExecutable.cmake` to match the macro defined in this file.

Reply:
Done

Reply:
I think you forgot to update the name in sycl/CMakeLists.txt